src/client/js/Widgets/BranchStatus/BranchStatusWidget.js

```diff
@@ -3,14 +3,16 @@
 
 /**
  * @author rkereskenyi / https://github.com/rkereskenyi
+ * @author pmeijer / https://github.com/pmeijer
  */
 
 define([
     'js/logger',
     'js/Controls/DropDownMenu',
     'js/Controls/PopoverBox',
     'js/Dialogs/Merge/MergeDialog',
-    'js/Constants'
+    'js/Constants',
+    'bootstrap-notify'
```

Comment:
does it not return with an object?
if we not use it, either comment it out or remove or add an 'unused' object to the function's parameter list